resources/artifact-ignores.properties

```
@@ -83,6 +84,7 @@ essentials                       # replaced by evergreen
 evergreen                        # not supposed to be installed manually. We use Incrementals http://repo.jenkins-ci.org/incrementals/io/jenkins/plugins/evergreen
 # deprecated
 external-scheduler = https://github.com/jenkins-infra/update-center2/pull/257
+fabric-beta-publisher = https://github.com/jenkins-infra/update-center2/pull/587
```

```suggestion
fabric-beta-publisher = https://github.com/jenkinsci/fabric-beta-publisher-plugin#deprecation-notice
```

Wouldn't saving one level of indirection be useful here? Or is this PR adding value on top of the notice? Could go either way. There's no particular reason to link to UC2 PRs other than those often being the best (or only) documentation available.